src/progress.ps1

```diff
@@ -0,0 +1,116 @@
+# U+2588	█	Full block
+# U+2589	▉	Left seven eighths block
+# U+258A	▊	Left three quarters block
+# U+258B	▋	Left five eighths block
+# U+258C	▌	Left half block
+# U+258D	▍	Left three eighths block
+# U+258E	▎	Left one quarter block
+# U+258F	▏	Left one eighth block
```

Comment:
Some of these characters do not render properly. My recommendation is we only use the full block, half block, and empty block.

Reply:
I haven't looked into this at all, but there is a PowerShell function available: [Write-Progress](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-progress). I'm guessing it is insufficient for whatever we are doing?

Reply:
It's insufficient in the sense that the aesthetics of it looks bad.

Reply:
@troyaf Should we remove the unicode values here that we aren't using?

Reply:
Done